nixos/modules/services/web-apps/limesurvey.nix

```diff
@@ -120,15 +120,15 @@ in
     };
 
     poolConfig = mkOption {
-      type = types.lines;
-      default = ''
-        pm = dynamic
-        pm.max_children = 32
-        pm.start_servers = 2
-        pm.min_spare_servers = 2
-        pm.max_spare_servers = 4
-        pm.max_requests = 500
-      '';
+      type = with types; attrsOf (oneOf [ str int bool ]);
```

Comment:
These changes seem to be backwards-incompatible. So should probably either make it compatible (by using `either lines (attrsOf ...)` or emit an error if the type is `lines` (e.g. with `coercedTo lines (x: throw "...") (attrsOf ...)` and a release note would be appropriate

Reply:
The `httpd` services (`limesurvey`, `mediawiki`, etc...) are brand new with no release yet. And I wrote them. As long as this goes through before `19.09` I suggest this is OK for these services.

Reply:
I see, sounds good then